tests/end-to-end/regression/5795/Issue5795Test.php

```diff
@@ -0,0 +1,27 @@
+<?php declare(strict_types=1);
+/*
+ * This file is part of PHPUnit.
+ *
+ * (c) Sebastian Bergmann <sebastian@phpunit.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace PHPUnit\TestFixture\Issue5795;
+
+use PHPUnit\Framework\TestCase;
+
+final class Issue5795Test extends TestCase
+{
+    /**
+     * @testWith		 [1]
+     * 			 [2]
+     * 			 [3]
+     * 
```

Comment:
for the test to fail this line needs to contain a space after the `*`.

my local phpstorm automatically strips this space and therefore I was not able to reproduce